src/ledger/LedgerTxnDataSQL.cpp

```diff
@@ -102,19 +119,25 @@ class BulkUpsertDataOperation : public DatabaseTypeSpecificOperation<void>
     void
     doSociGenericOperation()
     {
-        std::string sql = "INSERT INTO accountdata ( "
-                          "accountid, dataname, datavalue, lastmodified "
-                          ") VALUES ( "
-                          ":id, :v1, :v2, :v3 "
-                          ") ON CONFLICT (accountid, dataname) DO UPDATE SET "
-                          "datavalue = excluded.datavalue, "
-                          "lastmodified = excluded.lastmodified ";
+        std::string sql =
+            "INSERT INTO accountdata ( "
+            "accountid, dataname, datavalue, lastmodified, extension, "
+            "ledgerext "
+            ") VALUES ( "
+            ":id, :v1, :v2, :v3, :v4, :v5 "
+            ") ON CONFLICT (accountid, dataname) DO UPDATE SET "
+            "datavalue = excluded.datavalue, "
+            "lastmodified = excluded.lastmodified, "
+            "extension = excluded.extension, ledgerext "
+            " = excluded.ledgerext";
```

Comment:
not sure if this is a clang-format thing, but it would be nicer to have `ledgerext = excluded.ledgerext` on its own line

Reply:
Changed.  clang-format didn't disturb the change.